### ISSUE TITLE
tabs files

### ISSUE THREAD

**Issue by kevkha** [aldomatic/FB-Style-Page-Slide-Menu#7]
Thanks for the codes. What are the tabs files used for? I don't see them defined in index.html. Thanks.

**Comment by aldomatic** [comment]
A bit late on this, I forgot to remove that code and it doesn't need it. I will clean it up here soon.

**Comment by DemitryT** [comment]
Yeah, I noticed that too and just went with the code and script you included in your demo, which works perfectly. Until you have a chance to clean everything up, I would suggest putting a one liner in the README to tell people to copy what you do on the demo page, instead of looking at the js/css/index on the github page.
